.github/workflows/ccpp.yml

```diff
@@ -0,0 +1,48 @@
+name: Darling CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v2
+    - name: Checkout submodules
+      shell: bash
+      run: |
+        # If your submodules are configured to use SSH instead of HTTPS please uncomment the following line
+        # git config --global url."https://github.com/".insteadOf "git@github.com:"
+        git remote set-url origin https://github.com/darlinghq/darling.git
+        auth_header="$(git config --local --get http.https://github.com/.extraheader)"
+        git submodule sync --recursive
+        git -c "http.extraheader=$auth_header" -c protocol.version=2 submodule update --init --force --recursive --depth=1
+    - name: install
+      run: sudo apt-get install cmake clang bison flex libfuse-dev libudev-dev pkg-config libc6-dev-i386 linux-headers-generic gcc-multilib libcairo2-dev libgl1-mesa-dev libtiff5-dev libfreetype6-dev git libelf-dev libxml2-dev libegl1-mesa-dev libfontconfig1-dev libbsd-dev libxrandr-dev libxcursor-dev ccache devscripts debhelper ninja-build tree
+    - name: Prepare ccache timestamp
+      id: ccache_cache_timestamp
+      shell: cmake -P {0}
+      run: |
+        string(TIMESTAMP current_date "%Y-%m-%d-%H;%M;%S" UTC)
+        message("::set-output name=timestamp::${current_date}")
+    - name: ccache cache files
+      uses: actions/cache@v1.1.0
+      with:
+        path: /home/runner/.ccache
+        key: ${ { matrix.config.name } }-ccache-${ { steps.ccache_cache_timestamp.outputs.timestamp } }
+        restore-keys: |
+          ${ { matrix.config.name } }-ccache-
+    - name: ccache info
+      run: ccache -s
+    - name: make deb src
+      run: tools/makedeb --dsc
+    - name: make deb
+      run: tools/makedeb
+    - name: cp debs
+      run: cp ../*.deb debbuild/
+    - name: artifacts
+      uses: actions/upload-artifact@konradpabjan/v2-is-almost-here
+      with:
+        name: 'debs'
+        path: debbuild/*.deb
```

Comment:
Why not just upload the whole directory itself instead of using wildcards like ```path: debbuild```, also where is ```debbuild``` created?

Reply:
looks like just to 2 `deb`s by themselves are too large
https://github.com/stephengroat/darling/commit/c5b554465647f453f52be7f8b4c02cf8b1c6e51e/checks?check_suite_id=474725456

Reply:
Maybe try compressing them with:
```
- name: Move DEBs
  run: mkdir dist; cp ../*.deb dist
- name: Compress DEBs
  run: tar -cJf dist.tar.xz dist
- name: Upload Artifacts
  uses: actions/upload-artifact@v1
  with:
    name: 'debs'
    path: dist.tar.xz
```
Or just upload to Launchpad and not deal with Github storage limitations or do both!